k8s/upgrade/src/upgrade_resources/upgrade.rs

```diff
@@ -599,6 +607,7 @@ impl UpgradeResources {
                         Actions::Create,
                         skip_data_plane_restart,
                         upgrade_to_branch,
+                        upgrade_tag_for_test_envt,
```

Comment:
When is `upgrade_to_branch` used?

Reply:
it is used to upgrade to a branch , for testing , so this flag is hidden too. Right now with 2.1.0 release you can only upgrade from 2.0.0 or 2.0.1 to 2.1.0 . but this feature is there keeping future options in mind.